pkgs/applications/editors/vscode/default.nix

```diff
@@ -45,12 +47,40 @@ in
 
     desktopItem = makeDesktopItem {
```

Comment:
Should we just take the upstream one, and just replace the executable path?

Reply:
I thought of that, but I'd have to replace a lot of things
https://github.com/Microsoft/vscode/blob/9301aefe0f5b8c2505421ee57721e5bcfde2e117/resources/linux/code.desktop#L2